### ISSUE TITLE
HOLOCHAIN Development Node (Nixos based could be easiest, Mycelium inside)

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/tfgrid-sdk-ts#2016]

**Comment by Mahmoud-Emad** [comment]
Blocked on https://github.com/threefoldtech/tf-images/issues/203

**Comment by xmonader** [comment]
not a priority anymore
